app/Controllers/PaypalNotificationController.php

```diff
@@ -29,8 +29,10 @@ public function handle( FunFunFactory $ffFactory, Request $request ): Response {
 		try {
 			$ffFactory->getPayPalPaymentNotificationVerifier()->verify( $post->all() );
 		} catch ( PayPalPaymentNotificationVerifierException $e ) {
+			$parametersToLog = $post->all();
+			unset( $parametersToLog['payer_email'], $parametersToLog['payer_id'] );
```

Comment:
Maybe move the parameter keys that need to be removed to a class constant: `const BLOCK_LIST = [ 'payer_email', 'payer_id' ]`. Then loop them and unset here. That way we can modify it easier.
```suggestion
			foreach( BLOCK_LIST as $remove ) {
				unset( $parametersToLog[ $remove ] );
			}
```